web/templates/navbar.hbs

```diff
@@ -13,7 +14,9 @@
         <div class="column-middle" id="navbar-middle">
             <div class="column-middle-inner">
                 <div id="streamlist-toggle" class="tippy-zulip-delayed-tooltip {{#if embedded}}hide-streamlist-toggle-visibility{{/if}}" data-tooltip-template-id="show-left-sidebar-tooltip-template">
-                    <a class="left-sidebar-toggle-button" role="button" tabindex="0"><i class="fa fa-reorder" aria-hidden="true"></i>
+                    <a class="left-sidebar-toggle-button" role="button" tabindex="0">
+                        <i class="zulip-icon zulip-icon-panel-left" id="zulip-icon-panel-left" style="display: none;"></i>
```

Comment:
le's increase there font size to 18px and not use `style` attribute here but instead use CSS.